src/coreclr/pal/src/misc/fmtmessage.cpp

```diff
@@ -479,115 +383,27 @@ FormatMessageW(
                     nCount = 0;
                     goto exit;
                 }
-                if ( *lpSourceString == '!' )
```

Comment:
I'd like to see this fail in some way instead of silently not behaving like the windows FormatMessage api

Reply:
I thought about that. Wasn't sure of if a fail fast or assert or something else was appropriate. Thoughts?

Reply:
I'll error out like the other places in the function.